lib/mongo/events.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Mongo.Events.ServerDescriptionChangedEvent do
```

Comment:
I'm not sure we should make these records since they are a public API. In other parts of the public API, we use structs, and records seem to only be used internally. I'm assuming you wanted me to use records base on the assumption that it's internal-only, which isn't true, so I'll leave it as structs unless you have any objections.